content/Video_Display.md

```diff
@@ -670,10 +670,11 @@ sprite in OAM (\$FE00-\$FE03) has the highest priority, and so on. In
 Non-CGB mode, the smaller the X coordinate, the higher the priority. The
 tie breaker (same X coordinates) is the same priority as in CGB mode.
 
-The priority calculation between sprites disregards OBJ-to-BG Priority
-(attribute bit 7). Only the highest-priority nonzero sprite pixel at any
-given point is compared against the background. Thus if a sprite with a
-higher priority (based on OAM index) but with OBJ-to-BG Priority turned
+Priority among pixels that overlap is determined using the rules explained
```

Comment:
Priority is only considered among *opaque* pixels, though. An opaque pixel will always trump a transparent one, regardless of priority.